src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -653,6 +653,28 @@ def is_mapper_device(device_name):
     return device_name.startswith(('/dev/mapper', '/dev/dm-'))
 
 
+def is_writable_raw_device(disk_path):
```

Comment:
the function says `is_writable_raw_device` but the description says if the device can be opened in exclusive mode. Can we pick one description only? Or update the description in the docstring to convey that a writable device cannot be held by another process like a database.

Reply:
I had some semantic issues on that side. Fixed too.